DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/SimpleInjectorConfiguration.cs

```diff
@@ -29,7 +22,9 @@ public static Container RegisterPackages(Container container)
             container.RegisterPackages(new[]
             {
                 typeof(DALPackage).Assembly,
-                typeof(WebApiLibraryPackage).Assembly
+                typeof(WebApiLibraryPackage).Assembly,
+                typeof(NotesDALPackage).Assembly,
```

Comment:
I had some small problems with infrastructure for SI, but in the final version we agreed to simply add packages to one place and it should be one package for each project. In this case final version looks as follows: 
container.RegisterPackages(new[]
            {
                typeof(DALPackage).Assembly,
                typeof(WebApiLibraryPackage).Assembly,
                typeof(NotesDALPackage).Assembly,
                typeof(NotesWebApiPackage).Assembly
            });